web-local/tests/setup/setup.ts

```diff
@@ -22,7 +20,7 @@ setup("should prep projects", async () => {
  */
 async function prepProject(name: string) {
   const TEST_PROJECT_SRC_DIRECTORY = join(TEST_PROJECTS, name);
-  const TEST_PROJECT_DIRECTORY = join(BASE_PROJECT_DIRECTORY, name);
+  const TEST_PROJECT_DIRECTORY = join(TestTempDirectory, "projects", name);
```

Comment:
What's the difference between these two directories? It seems like we should just have one of these constants?